### PR TITLE
Adding new api for cci data

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: userprofile/views.py [hunk 1]
```diff
@@ -149,3 +149,14 @@ def get_object(self):
...
+
+class HealthCheckCciData(GenericViewSet, ListModelMixin):
+    serializer_class = HealthCheckUserProfileSerializer
+
+    def post(self, request):
+        # data will go here to call the task
+
+        # get user screening data
+        data = request.data
+        return Response(data=data, status=status.HTTP_200_OK)
```
Comment (erikh360) [inline userprofile/views.py]: We don't really need to return the data back to Rapidpro

**Review by erikh360 — COMMENTED** [review]
As per comment in slack, please move to tbconnect and rename appropriately. Otherwise looks good

File: tbconnect/tests/test_views.py [hunk 2]
```diff
@@ -460,3 +460,12 @@ def test_reset_allowed(self):
         self.client.force_authenticate(user)
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class HealthCheckCciDataTest(APITestCase):
```
Comment (erikh360) [inline tbconnect/tests/test_views.py]: Lets rename this to match too

File: tbconnect/views.py [hunk 3]
```diff
@@ -76,3 +76,14 @@ def destroy(self, request, pk=None):
             except TembaNoSuchObjectError:
                 continue
         return Response({"status": "OK"})
+
+
+class TBCheckCciDataViewSet(GenericViewSet, ListModelMixin):
```
Comment (erikh360) [inline tbconnect/views.py:81]: Do we need this `ListModelMixin`?

File: tbconnect/views.py [hunk 4]
```diff
@@ -76,3 +76,14 @@ def destroy(self, request, pk=None):
...
+    serializer_class = TBCheckSerializer
```
Comment (erikh360) [inline tbconnect/views.py]: I assume this is just a placeholder serializer until we write the cci one

**Review by erikh360 — COMMENTED** [review]
Nice, few more small comments

File: tbconnect/views.py (hunk 3 above)
Comment (Buhle79) [reply tbconnect/views.py:81]: Will investigate later.

**Review by erikh360 — APPROVED** [review]

**Pull request merged**